src/coprocessor/codec/mysql/mod.rs

```diff
@@ -15,7 +15,7 @@ use super::Result;
 use util::escape;
 
 /// `UN_SPECIFIED_FSP` is the unspecified fractional seconds part.
-const UN_SPECIFIED_FSP: i8 = -1;
+pub const UN_SPECIFIED_FSP: i8 = -1;
```

Comment:
I think both are ok here. Since this PR is too large, maybe we could change it in an independent PR someday? @hicqu